### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by fda77**
open menu at start
framework: show reboot items as icons
modules: indexer&scroller
download: refesh icon
settings,support about as fragmens
dont remove notification at framework&modules
.. some more i forgot?
known bug: settings PreferencesFragment ha sno correct topmargin, but 2 invisible preferences "fixes" it

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -3,7 +3,11 @@
     package="de.robv.android.xposed.installer"
     xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <uses-permission android:name="android.permission.ACCESS_SUPERUSER"/>
+    <uses-sdk
```
Comment (DVDAndroid) [inline app/src/main/AndroidManifest.xml]: Why are you declaring uses-sdk here when is already declared in build.gradle?
And why min-sdk is 21? This app has support KitKat also, but not the framework installation

**Comment by DVDAndroid** [comment]
Thanks for this pull request. Before merging it in the master branch, I need to test this changes

**Comment by fda77** [comment]
Take you time, many changes.
Maybe you find a better way as the workaround in settings!
Is it expected to open 30 installer views on "import"? It surprised me a little.
Has KK support? Oh sry :D  I'm a Eclips & SVN user so i dont know about the gradle..
ACCESS_SUPERUSER is deprected
Please also see https://github.com/fda77/XposedInstaller/commit/4adc6bf09b312c5d145b32cb0612b0f216ed68de which was reverted in https://github.com/fda77/XposedInstaller/commit/c80d2d3d00dc2468a920e2f9e0f946ddd20919e2 before converted activities to fragments

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (DVDAndroid) [inline app/src/main/AndroidManifest.xml:6]: Remove these lines please

Comment (fda77) [inline app/src/main/AndroidManifest.xml:6]: As i said im not experienced with git(hub). can't you just take the lines you like?

**Comment by DVDAndroid** [comment]
I'm sorry but I won't merge this pull request in the master branch.

Things I like:
- open menu at start
- modules: indexer&scroller
- padding for small fingers

Things that I don't like:
- framework: show reboot items as icons
- settings,support about as fragmens
- workaround in preferencefragment

Things that I'll improve:
- notification that disappear immediately in modules fragment
- log saving in real storage

Now, if you want, you can create some pull request with things I like, for example, list scroller & indexer

**Pull request closed without merging**